src/components/Board/services/index.js

```diff
@@ -75,4 +75,40 @@ function removeCard(board, fromLane, card) {
   return { ...board, lanes: filteredLanes }
 }
 
-export { moveLane, moveCard, addLane, removeLane, renameLane, addCard, removeCard }
+function getCoordinates(event) {
```

Comment:
It'll be exported along with the helper methods. What do you think to move the helper to the `src/services/helpers.js`?